pkg/common/oras/authprovider/azure/azureworkloadidentity.go

```diff
@@ -130,9 +130,23 @@ func (d *azureWIAuthProvider) Provide(ctx context.Context, artifact string) (pro
 	serverURL := "https://" + artifactHostName
 
 	// create registry client and exchange AAD token for registry refresh token
-	refreshTokenClient := containerregistry.NewRefreshTokensClient(serverURL)
+	// TODO: Consider adding authentication client options for multicloud scenarios
+	client, err := azcontainerregistry.NewAuthenticationClient(serverURL, nil) // &AuthenticationClientOptions{ClientOptions: options})
```

Comment:
suggestion to make a variable ( improved readability for why the param is nil) , assign it to nil then pass into NewAuthenticationClient